specification/cosmos-db/resource-manager/Microsoft.DocumentDB/2015-04-08/cosmos-db.json

```diff
@@ -437,26 +437,432 @@
+          },
+          {
+            "$ref": "#/parameters/databaseRidParameter"
+          },
+          {
+            "$ref": "#/parameters/collectionRidParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The metrics for the database account were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/MetricListResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/region/{region}/metrics": {
+      "get": {
+        "operationId": "DatabaseAccountRegion_ListMetrics",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountRegionGetMetrics": { "$ref": "./examples/CosmosDBDatabaseAccountRegionGetMetrics.json" }
+        },
+        "description": "Retrieves the metrics determined by the given filter for the given database account and region.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/regionParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/filterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The metrics for the database account were retrieved successfully.",
+            "schema": {
+              "$ref": "#/definitions/MetricListResult"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/usages": {
+      "get": {
+        "operationId": "DatabaseAccounts_ListUsages",
+        "x-ms-examples": {
+          "CosmosDBDatabaseAccountGetUsages": { "$ref": "./examples/CosmosDBDatabaseAccountGetUsages.json" }
+        },
+        "description": "Retrieves the usages (most recent data) for the given database account.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/accountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/usageFilterParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
okay, same here, please review all added methods 😉

Reply:
same--not going to be multiple pages